Python/Product/TestAdapter/visualstudio_py_testlauncher.py

```diff
@@ -239,7 +240,9 @@ def main():
     (opts, _) = parser.parse_args()
     
     sys.path[0] = os.getcwd()
-    
+    if(opts.debugger_path is not None):
```

Comment:
Remove parentheses

Reply:
Also, if it's empty, it probably shouldn't be added to `sys.path`, so you can remove the `is not None` part.